common/lib/xmodule/xmodule/js/src/html/edit.js

```diff
@@ -171,7 +172,37 @@
            */
           init_instance_callback: this.initInstanceCallback,
           browser_spellcheck: true
-        });
+        };
+
+        if (process.env.ADDITIONAL_NODE_ENV_VARS) {
+          var tinyMceAdditionalPlugins = JSON.parse(process.env.ADDITIONAL_NODE_ENV_VARS).TINYMCE_ADDITIONAL_PLUGINS;
```

Comment:
This seems a bit strange. I would expect that if I set `ADDITIONAL_NODE_ENV_VARS = {"foo": "bar"}` in the platform configuration, then I would be able to read it as `process.env.foo`

In other words, I would expect that a key set in a variable called "additional node environment variables" would become a node environment variable.

Maybe just rename `ADDITIONAL_NODE_ENV_VARS` to `JS_ENV_EXTRA_CONFIG` or something?

Reply:
Yeah, you have a good point. Thanks for the suggestion which I went with, it made it easier to make the change. :+1: